mount_windows.go

```diff
@@ -0,0 +1,32 @@
+package containerd
+
+import (
+	"errors"
+)
+
+// Mount is the lingua franca of containerd. A mount represents a
+// serialized mount syscall. Components either emit or consume mounts.
+type Mount struct {
```

Comment:
This type can be shared between the two implementations. Just move the method to another file.

I think this works. Let me know if I'm way off.